posthog/api/decide.py

```diff
@@ -93,6 +93,10 @@ def get_base_config(token: str, team: Team, request: HttpRequest, skip_db: bool
 
     REMOTE_CONFIG_CACHE_COUNTER.labels(result=use_remote_config).inc()
     surveys_opt_in = get_surveys_opt_in(team)
+    if surveys_opt_in:
+        surveys_count = get_surveys_count(team)
+        if surveys_count <= 0:
+            surveys_opt_in = False
```

Comment:
```suggestion
    surveys_opt_in = get_surveys_opt_in(team) and get_surveys_count(team) > 0
```

what do you think about this intead? since we're not really using the `surveys_count` variable anywhere else

Reply:
yes, makes sense